templates/stamp/utilities.sh

```diff
@@ -1249,12 +1310,8 @@ is_master_server()
     third_server=${replicated_servers_list[$third_server_position]}
 
     # more defensive: the repladmin will override the file/or create a new file
-    replication_status_file="/tmp/replication_status_$server.csv"
-    if [[ -f $replication_status_file ]];
-    then
-        #log "Cleaning up existing replication file"
-        rm $replication_status_file
-    fi
+    replication_status_file="$(mktemp /tmp/replication_status_XXXXXX.csv)"
+    remove_replication_file "${replication_status_file}"
```

Comment:
nit: we shouldn't need to clean it up beforehand. the > operator used later clean the file. this is an optional suggestion

Reply:
It's more defensive than not.